admin_guide/router.adoc

```diff
@@ -17,14 +17,56 @@ xref:../architecture/networking/routes.adoc#architecture-core-concepts-routes[ro
 topic discusses the HAProxy template router and the components to check to
 ensure its health.
 
+[[admin-guide-router-view-stats]]
 == Viewing Statistics
+
 The HAProxy router exposes a web listener for the HAProxy statistics. Enter the
 router's public IP address and the correctly configured port (*1936* by default)
-to view the statistics page, and enter the administrator password when prompted.
-This password and port are configured during the router installation, but they
+to view the statistics page. The administrator password and port are configured 
+during the router installation, but they
 can be found by viewing the *_haproxy.config_* file on the container.
 
+To view HAProxy router stats:
+
+. If needed, xref:../install_config/router/index.html#install-config-router-overview[create the router] 
+using `--stats-port=1936` to expose statistics on the specified port: 
++
+----
+$ oadm router router --replicas=1  --selector='zone=west' --stats-port=1936
+----
++
+[NOTE]
+====
+`-replicas` is the replication factor of the router; commonly 2 when high availability is desired.
+
+`-selector` is used to filter nodes on deployment. Used to run routers on a specific set of nodes.
+====
+
+. If needed, run the following command to get the stats password that was assigned during router creation:
++
+----
+$ oc describe pod <pod-name> |grep STATS_PASSWORD
+----
+
+. Run the following command to open iptables for the router stats port:
++
+----
+$ iptables -I OS_FIREWALL_ALLOW -p tcp -m tcp --dport 1936 -j ACCEPT
+----
++
+You can add this rule to *_/etc/sysconfig/iptables_* to keep the rule across reboots.
++
+You can make this port accessible via xref:../dev_guide/port_forwarding.adoc#dev-guide-port-forwarding[port forwarding], if desired. 
+
+. Use the URL in a browser to launch the stats window:
++
+----
+http://admin:<stats-password>@<master-ip>:1936/haproxy_stats
```

Comment:
path /haproxy_stats has been deprecated from 3.6
cc@knobunc

Reply:
@knobunc ^^

Reply:
We should encourage people to use the prometheus stats.  @pecameron we need to make sure that we re-sync when we work out what the oadm command ends up looking like.

Reply:
BTW the block at line 115 should be put in its own section.  How about "Retrieving HAProxy Stats Locally"

That's the usual way that I grab the stats... it always works.